projects/fbtc/index.js

```diff
@@ -1,24 +1,13 @@
-const { getConfig } = require('../helper/cache')
 const { sumTokens } = require('../helper/chain/bitcoin')
 const sdk = require('@defillama/sdk')
-const { get } = require('../helper/http')
-const { getEnv } = require('../helper/env')
+const bitcoinAddressBook = require('../helper/bitcoin-book/index.js')
 
 const abi = {
   getQualifiedUserInfo: 'function getQualifiedUserInfo(address _user) view returns ((bool locked, string depositAddress, string withdrawalAddress) info)',
 }
+
 async function tvl() {
-  const staticAddresses = await getConfig('fbtc', undefined, {
-    fetcher: async () => {
-      const token = getEnv('FBTC_ACCESS_TOKEN')
-      const { result } = await get('https://fbtc.phalcon.blocksec.com/api/v1/extension/fbtc-reserved-addr', {
-        headers: {
-          'access-token': token
-        }
-      })
-      return result.map(r => r.address)
-    }
-  })
+  const staticAddresses = await bitcoinAddressBook.fbtc()
```

Comment:
this fetcher needs to be updated to include the next few lines to get complete list of addresses